go.mod

```diff
@@ -60,7 +60,7 @@ require (
 	github.com/magiconair/properties v1.8.1 // indirect
 	github.com/mailru/easyjson v0.7.1 // indirect
 	github.com/mattermost/go-i18n v1.11.0
-	github.com/mattermost/gorp v2.0.1-0.20200527092429-d62b7b9cadfc+incompatible
+	github.com/mattermost/gorp v1.6.2-0.20200526162043-8cc0a796020f
```

Comment:
Don't know why master is 1.6.2 but the current branch, which is below master, is 2.0.1 :\

Reply:
The pseudo-version should guarantee the right behaviour here, but agree that the tag prefix is often confusing. It's /usually/ because the tag contains a commit that's newer than the version in question, but I can't find evidence of that here.

Probably good to do a file-by-file comparison with the expected changes in the `vendor` package to be 100% certain you're getting what you want.

Reply:
I see. Ok, I'll review the vendor package to verify is consistent

Reply:
Checked file by file and everything seems ok with the vendor files

Reply:
As discussed offline, should we just make a v2 of gorp to align the tag version and go.mod version?

Reply:
We could talk about this after the merge but is possible that we need to do it for our own fork